packages/fxa-content-server/webpack.config.js

```diff
@@ -62,8 +62,9 @@ const webpackConfig = {
       draggable: require.resolve('jquery-ui/ui/widgets/draggable'),
       duration: require.resolve('duration-js/duration'),
       'fast-text-encoding': require.resolve('fast-text-encoding'),
-      fxaCryptoDeriver: require.resolve(
-        'fxa-crypto-relier/dist/fxa-crypto-relier/fxa-crypto-deriver'
+      fxaCryptoDeriver: path.resolve(
+        __dirname,
+        '../../dist/libs/vendored/crypto-relier/src/index.js'
```

Comment:
I tried using `require.resolve('@fxa/vendored/crypto-relier')` would work but it didn't. Maybe I'm missing something?

Reply:
`@fxa` is not an alias that the node module resolver would know about unless there's some magic I'm not aware of.

Reply:
There's magic that I used in 123done, not sure if it'd work here but there's alias packages for various use-cases that make it work, e.g. I had to set this up:
https://github.com/mozilla/fxa/commit/9a649c6441a33823c9237956582b2bf08ac23cc9#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R265

And to get 123done to use it:
https://github.com/mozilla/fxa/commit/9a649c6441a33823c9237956582b2bf08ac23cc9#diff-82cb871fa708015fd2a5e89be112959108a1839c19d3cc28b7560f2a38f911aeR1

I'm not sure if module-alias works for webpack, but perhaps there's something similar if it doesn't?